USAGE.md

```diff
@@ -32,75 +39,106 @@ Example bucket policy:
+export AWS_ACCESS_KEY_ID=$(jq -r ".Credentials[\"AccessKeyId\"]" <<< "$credentials")
+export AWS_SECRET_ACCESS_KEY=$(jq -r ".Credentials[\"SecretAccessKey\"]" <<< "$credentials")
+export AWS_SESSION_TOKEN=$(jq -r ".Credentials[\"SessionToken\"]" <<< "$credentials")
 ```
 
-# Data Lake Lambda Endpoints Usage
-## Endpoint Request Format
-```
+## Dataset format
+
+A dataset consists of a set of files in Amazon Web Services Simple Storage Service (AWS S3). The dataset consists of geospatial metadata files in [SpatioTemporal Asset Catalogs (STAC)](https://stacspec.org/) format and data files, which are called "assets" in STAC.
+
+The GDL performs many checks on datasets. If any of the checks fail the dataset will not be imported, so it's important to know what they are. The following list is a reference of all the checks which are currently in place.
+
+- Every metadata file must follow the [STAC Collection Specification](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md).
+- Every metadata URL (in the [`links` property](https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#link-object)) must be an S3 URL of the form `s3://BUCKET_NAME/KEY`, for example, `s3://my-bucket/some-path/foo.tif`.
+- Every asset (in the [`item_assets` property](https://github.com/radiantearth/stac-spec/blob/master/extensions/item-assets/README.md)) must have:
+   - an S3 [URL](https://github.com/radiantearth/stac-spec/blob/master/item-spec/item-spec.md#asset-object), as defined above
+   - a [multihash](https://github.com/radiantearth/stac-spec/blob/master/extensions/checksum/README.md) corresponding to the contents of the asset file
+- Every metadata and asset must be in the same S3 "folder." That is, every metadata and asset URL must have the same content up to the last slash. For example, a having a root metadata URL `s3://bucket/folder/collection.json` and an asset URL `s3://bucket/folder/subfolder/1.tif` is invalid.
+- Every metadata and asset URL must be readable by the GDL.
+- A dataset *may* refer to the same asset more than once. All references to the same asset must have the same multihash. That is, having a SHA-1 and a SHA-256 checksum for the same file will be considered invalid, even if both checksums are valid. This is to enable a simpler checksum validation.
+
+## Data Lake Endpoints Usage
+
+There are two end user interaction points in GDL:
+
+- A [datasets endpoint](TODO), to create, get, update or delete individual datasets, and to list all datasets
+- A [dataset versions endpoint](TODO), to create new versions of datasets. The S3 files which constitute the dataset are all linked to a specific dataset version.
+
+Both of these are implemented as AWS Lambda jobs, which means they can be run ("invoked") either via the AWS web interface (links above) or via any tool using the AWS API, such as the commands below.
```

Comment:
It is Lambda "function" not "job". Also what do you mean by "(links above)" ? Which links ?

Reply:
Fixed. I meant the links *just* above this paragraph, which are currently marked "TODO".